### DISCUSSION TITLE
`:zip` and `:tar` commands exit status 1

### DISCUSSION THREAD

**Discussion by NunonuN** [gokcehan/lf#1144] [Q&A] [answered]
When trying to compress a file using the `:zip` or `:tar` commands, I get the following error message:

`running shell: exit status 1`

After exiting **lf**, I can see the following message printed on the terminal:

`--: line 4: $1: unbound variable` (when interpreter is **sh**)
`zsh:4: 1: parameter not set` (when interpreter is **zsh**)

It seems that the commands do not capture `$1`. This happens whether I select one or more files or I just issue the command over any file.
How can this be solved?

**Comment by marendowski** [comment]
Hi, I fixed it with mapping it like that: `map mz push :zip<space>` instead of normal `map mz zip`.

**Comment by og900aero** [comment, marked as answer]
In lfrc you must use this:

```
cmd zip ${{
    set -f
    clear
	zip -r $1.zip $fx
    lf -remote "send unselect"
}}
```
Reload lf (for apply new config) select what you want to compress and type:
:zip filename (.zip extension is not necessary)
